src/replica/replica_stub.cpp

```diff
@@ -335,6 +335,61 @@ void replica_stub::install_perf_counters()
         "write size exceed threshold count in the recent period");
 
+    // <- Bulk Load Metrics ->
+
+    _counter_bulk_load_running_count.init_app_counter("eon.replica_stub",
+                                                      "bulk.load.running.count",
+                                                      COUNTER_TYPE_VOLATILE_NUMBER,
+                                                      "current bulk load running count");
+    _counter_bulk_load_downloading_count.init_app_counter("eon.replica_stub",
+                                                          "bulk.load.downloading.count",
+                                                          COUNTER_TYPE_VOLATILE_NUMBER,
+                                                          "current bulk load downloading count");
+    _counter_bulk_load_ingestion_count.init_app_counter("eon.replica_stub",
+                                                        "bulk.load.ingestion.count",
+                                                        COUNTER_TYPE_VOLATILE_NUMBER,
+                                                        "current bulk load ingestion count");
+    _counter_bulk_load_succeed_count.init_app_counter("eon.replica_stub",
+                                                      "bulk.load.succeed.count",
+                                                      COUNTER_TYPE_VOLATILE_NUMBER,
+                                                      "current bulk load succeed count");
+    _counter_bulk_load_failed_count.init_app_counter("eon.replica_stub",
+                                                     "bulk.load.failed.count",
+                                                     COUNTER_TYPE_VOLATILE_NUMBER,
+                                                     "current bulk load failed count");
+    _counter_bulk_load_ingestion_reject_write_count.init_app_counter(
+        "eon.replica_stub",
+        "bulk.load.ingestion.reject.write.count",
+        COUNTER_TYPE_VOLATILE_NUMBER,
+        "bulk load ingestion reject write requests count");
+    _counter_bulk_load_download_file_succ_count.init_app_counter(
+        "eon.replica_stub",
+        "bulk.load.download.file.success.count",
+        COUNTER_TYPE_VOLATILE_NUMBER,
+        "bulk load recent download file success count");
+    _counter_bulk_load_download_file_fail_count.init_app_counter(
+        "eon.replica_stub",
+        "bulk.load.download.file.fail.count",
+        COUNTER_TYPE_VOLATILE_NUMBER,
+        "bulk load recent download file failed count");
+    _counter_bulk_load_download_file_size.init_app_counter("eon.replica_stub",
+                                                           "bulk.load.download.file.size",
+                                                           COUNTER_TYPE_VOLATILE_NUMBER,
+                                                           "bulk load recent download file size");
+    _counter_bulk_load_max_download_file_size.init_app_counter("eon.replica_stub",
```

Comment:
What is a "max" file size used for? Why not calculate this value by `COUNTER_TYPE_PERCENTILE`?

Reply:
It is used to record max downloaded file size, just to show the maximum the size. It seems that it is a little meaningless.